app/src/main/res/values/strings.xml

```diff
@@ -68,4 +68,21 @@
 
     <string name="my_schedule_empty">No items here. Let\'s check out all sessions and put your favourite!</string>
     <string name="my_schedule_go_to_session_list">Go to All Sessions</string>
+
+    <string name="session_feedback_relevance">How relevant was this session to your projects?</string>
+    <string name="session_feedback_not_at_all_relevant">Not at all</string>
+    <string name="session_feedback_extremely_relevant">Extremely</string>
+    <string name="session_feedback_knowledgeable">How knowledgeable was the content?</string>
+    <string name="session_feedback_poor_knowledge">Poor</string>
+    <string name="session_feedback_outstanding_knowledge">Outstanding</string>
+    <string name="session_feedback_expectation_label_text">Based on the description, was the content as expected?</string>
```

Comment:
@ken5scal Sorry. I noticed it might be confusing as we already used "relevalnt" in a previous question. Do you have any idea on how to say this with a word "as expected"?